class-es-wp-query-wrapper.php

```diff
@@ -614,7 +614,12 @@ public function get_posts() {
 
 		// Author/user stuff
 		if ( ! empty( $q['author'] ) && $q['author'] != '0' ) {
-			$q['author'] = addslashes_gpc( '' . urldecode( $q['author'] ) );
+			if ( is_array( $q['author'] ) ){
```

Comment:
Under what circumstance would `$q['author']` be an array? According to the codex and the inline docs, this should only ever be an int or a string